### PR TITLE
Init website content

### PR DESCRIPTION
**Pull request by leonardpahlke**
There is still a lot of content missing, but I have updated the main page (which should be ready) and initialized the site tab structure :)

cc @caradelia

### CODE REVIEW

**Comment by leonardpahlke** [comment]
ref tracking issue: https://github.com/cncf/tag-env-sustainability/issues/53

**Review by cjyabraham — APPROVED** [review]
LGTM

File: website/content/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
-title: "TAG Environmental Sustainability"
-toc_hide: true
+title: Events
```
Comment (cjyabraham) [inline website/content/_index.md:2]: I assume you didn't want to change the homepage title to "Events"?

File: website/content/events/_index.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+---
+title: Events TAG Environmental Sustainability
```
Comment (cjyabraham) [inline website/content/events/_index.md:2]: I assume this should be just "Events"?

**Comment by cjyabraham** [comment]
After merging, I noticed a few things that looked off. I'll fix them in the code.

**Pull request merged**